jmespath/jputil/util_test.go

```diff
@@ -0,0 +1,20 @@
+package jputil
```

Comment:
I'd suggest adding some more test for this package.  specifically, IsFalse, ObjEquals, ToArrayNum, and ToArrayStr. Those functions aren't really tested.